index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">  
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">  
+    <title>Survey Form</title>
+</head>
+<body>
+    <h1 id="title">freeCodeCamp Survey Form</h1>
+    <p id="description">Thank you for taking the time to help us improve the platform</p>
+    <form action=" hello" id="survey-form">
+        <p>Name</p>
+        <input type="text" placeholder="Enter your name" id="name">
+        <p>Email</p>
+        <input type="text" placeholder="Enter your Email" id="email">
+        <p>Age</p>
+        <input type="number" placeholder="Enter your age" id="number" min="16" max="100">
+        <p>Which option best describes your current role?</p>
+        <select name="name1" id="select1"><option label="label1" value="Student">Student</option><option value="Full time job">Full time job</option></select>
+
+
```

Comment:
Be careful with blank spaces, It could be confusing for other people. 😖  And it isn't considered as a best practices

Reply:
- [ ] The way you start this sentence is not really friendly although what you want to mention is interesting, it can be hurtful for the person you are reviewing. Remember that people put a lot of effort into their work before submitting for reviews so it would be good to be empathic with them even though you have a lot to say about their work.

Reply:
Got it!
Thanks!